src/init/react-tanstack/src/routes/__root.tsx

```
@@ -0,0 +1,87 @@
+function RootComponent() {
+  return (
+    <RootDocument>
+      <Outlet />
+    </RootDocument>
+  );
+}
+
+function RootDocument({ children }: Readonly<{ children: ReactNode }>) {
+  return (
+    <html>
+      <head>
```

Biome (2.1.2)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)